### DISCUSSION TITLE
Importar Arquivos Gif

### DISCUSSION THREAD

**Discussion by ESTUQUE** [Sketchware-Pro/Sketchware-Pro#639] [Ideas]
Peço que tornem possível importar arquivos gif
Esse video da YouTube mostra um método,  problema, é que este método não permite importar mais arquivos de som!
https://youtu.be/8JuEWYRaszs

Please make it possible to import gif files
 This YouTube video shows a method, problem is that this method no longer allows you to import sound files!
 https://youtu.be/8JuEWYRaszs

**Comment by j-px** [comment]
That's a pretty hackish way to add GIFs to projects. Don't do that, use Resource Manager instead. Create a `raw` folder there, and add your .gif file there. The same code to load the .gif animation should work.

**Reply by ESTUQUE** [reply]
Parece fácil a explicação, mas não entendi muita coisa😅
Dá pra a ser mais específico,  sobre a pasta `raw`

**Reply by j-px** [reply]
Here's screenshots for that:

![Screenshot_20221115_193058_Sketchware Pro.png](https://user-images.githubusercontent.com/40118124/201998464-04d8b8ee-ef80-401c-9a19-7ec2a5dfc663.png)  
![Screenshot_20221115_193212_Sketchware Pro.png](https://user-images.githubusercontent.com/40118124/201998515-075f8e74-f352-49fe-aeeb-5e68af5a95ca.png)  
![Screenshot_20221115_193220_Sketchware Pro.png](https://user-images.githubusercontent.com/40118124/201998540-fab682e6-1268-450f-a5ef-3a14336b9c81.png)

**Reply by ESTUQUE** [reply]
wow!  Thank you very much it worked!  The file was named gif.
`url = "android.resource://"+getPackageName()+"/raw/gif";
Glide.with(getApplicationContext()).load(Uri.parse(url)).into(imageview1);`
Thanks
Thanks!!

**Reply by ESTUQUE** [reply]
If you can, help me to define this `gif` or `url` in the background of this `imageview1`.

**Reply by j-px** [reply]
What do you mean? The code you sent earlier should work already.

**Reply by ESTUQUE** [reply]
O código está funcionar correctamente com o método que você me mostrou, tem como definir a mesma imagem gif no background de uma layout?

**Reply by j-px** [reply]
I don't really, sorry.
